example/vagrant_box_example/main.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_providers {
+    nomad = {
+      version = ">=1.4.0"
```

Comment:
We might need to update it to the latest. https://github.com/hashicorp/terraform-provider-nomad/releases/tag/v1.4.9 

Not sure that in current PR

Reply:
yeah..you are right! Created a separate issue to handle it : https://github.com/fredrikhgrelland/terraform-nomad-minio/issues/30